feature/#450 added doc for get_entities_by_config_id (#443)
Avaiga/taipy-doc: docs/manuals/core/entities/data-node-mgt.md

    # data_node == data_node_retrieved
    ```

> # Get data nodes by config_id

Reviewer: To be consistent with the other changes, you should remove the underscore.

The data nodes that are part of a **scenario**, **pipeline** or **task** can be directly accessed as attributes by

> using its config_id:

Reviewer: "using its config_id" -> "using their configuration identifier:"

Reviewer 2: I am not sure about replacing `config_id` with "configuration identifier" is good for the understanding. 
On the one hand, it s better English, I agree with that, but on the other hand, `config_id` is the name of the attribute. So it s easier to understand.

I would personally go for the name of the attribute name (with the appropriate styling like between \` or \_ ) for a better understanding.

Author: I agree with JR here, `config_id` to me is straight-forward that's (to my understanding) being used in multiple places within the doc :D

    task.sales_history
    ```

> Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^`, which will return

Reviewer 3: ```suggestion
Data nodes can also be retrieved by calling `taipy.get_entities_by_config_id()^` with their config_id, which will return
```

to call a method with a parameter.

This can also be applied to the pipeline, scenario, task phrases below.

> Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^`, which will return
> a list of existing data nodes based on the config_id.

Reviewer 2: ```suggestion
Data nodes can also be retrieved from their config_id using `taipy.get_entities_by_config_id()^`.

It returns the list of all existing data nodes instantiated from the config_id provided as a parameter.
```

!!! Example

    ```python linenums="1"
    import taipy as tp
    import my_config

>     scenario = tp.create_scenario(my_config.monthly_scenario_cfg)

Reviewer 2: Maybe we can instantiate 2 scenarios in the example and explain that calling `tp.get_entities_by_config_id("trained_model")` returns 2 data nodes.

    # Get all trained model data nodes by config_id

>     all_trained_model_dns = tp.get_entities_by_config_id(my_config.trained_model_cfg.id)

Reviewer 3: ```suggestion
    all_trained_model_dns = tp.get_entities_by_config_id("trained_model")
```

Should we represent the config_id here as a string?
It is easier to understand imo.

    task.sales_history
    ```

> Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.

Reviewer: "by their config_id" -> "using their configuration identifier"

> It returns the list of all existing data nodes instantiated from the config_id provided as a parameter.

Reviewer: "It" -> "This method"
"config_id provided as a parameter" -> "indicated configuration identifier"
